clustered/server/entity/src/main/java/org/ehcache/clustered/server/store/ClusterTierPassiveEntity.java

```diff
@@ -191,6 +191,7 @@ private EhcacheEntityResponse invokePassiveInternal(InvokeContext context, Ehcac
     return success();
   }
 
+  @SuppressWarnings("deprecation")
```

Comment:
this is ok to leave all these deprecations ?

Reply:
@myronkscott and I had a chat about them and he promises that he'll fix them in the next push.